app/controllers/likes_controller.rb

```diff
@@ -0,0 +1,36 @@
+class LikesController < ApplicationController
+  before_action :find_specific_post, only: [:like]
+
+  def new
+    @like = Like.new
+  end
+
+  def create
+    return if liked?
+
+    redirect_to "/users/#{params[:user_id]}"
+
+    Like.create(like_params)
+    redirect_to user_post_path(user_id: params[:user_id], id: params[:post_id]), notice: 'Like Added!'
+
+    # @post.likes.create(like_params)
+    # redirect_to user_post_path(
+    #   user_id: params[:user_id],
+    #   id: params[:post_id]
+    # )
```

Comment:
-  [ ] It would be nice to rid the codebase of any unused lines of code to make it look clean. Please handle this.